src/site/_includes/partials/translation-disclaimer.njk

```diff
@@ -8,8 +8,8 @@
     <!-- Must be an absolute url to enforce language reload.  -->
     <a href="/en{{ canonicalUrl }}" target="_blank">the original post</a>.
     {% endif %}
-    {% if gitlocalizeUrl %}
+    {# {% if gitlocalizeUrl %}
```

Comment:
Did you mean to comment this out?

Reply:
Yes, until we resolve the process for processing community translations.